<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmLiquidacion</class>
 <widget class="QMainWindow" name="FrmLiquidacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>644</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Liquidación</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabdetails">
       <attribute name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/res/document-edit.png</normaloff>:/icons/res/document-edit.png</iconset>
       </attribute>
       <attribute name="title">
        <string notr="true"/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Datos de declaración</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>No. Poliza</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtPolicy</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" colspan="3">
                <widget class="QLineEdit" name="txtPolicy">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Proveedor</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1" colspan="3">
                <widget class="QStackedWidget" name="swProvider">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="page">
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLineEdit" name="txtProvider">
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="page_2">
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QComboBox" name="cbProvider">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="editable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Fecha Poliza</string>
                 </property>
                 <property name="buddy">
                  <cstring>dtPicker</cstring>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDateEdit" name="dtPicker">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>TC</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtExchangeRate</cstring>
                 </property>
                </widget>
               </item>
               <item row="2" column="3">
                <widget class="QLineEdit" name="txtExchangeRate">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>83</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Procedencia</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtSource</cstring>
                 </property>
                </widget>
               </item>
               <item row="3" column="1" colspan="3">
                <widget class="QLineEdit" name="txtSource">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Bodega</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1" colspan="3">
                <widget class="QStackedWidget" name="swWarehouse">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="currentIndex">
                  <number>1</number>
                 </property>
                 <widget class="QWidget" name="page_3">
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <item>
                    <widget class="QComboBox" name="cbWarehouse">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="editable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="page_4">
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLineEdit" name="txtWarehouse">
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0" colspan="2">
                <layout class="QFormLayout" name="formLayout_2">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetMinAndMaxSize</enum>
                 </property>
                 <property name="fieldGrowthPolicy">
                  <enum>QFormLayout::FieldsStayAtSizeHint</enum>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Agencia</string>
                   </property>
                   <property name="buddy">
                    <cstring>sbAgency</cstring>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDoubleSpinBox" name="sbAgency">
                   <property name="minimumSize">
                    <size>
                     <width>157</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="prefix">
                    <string>C$ </string>
                   </property>
                   <property name="maximum">
                    <double>999999999.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Almacen</string>
                   </property>
                   <property name="buddy">
                    <cstring>sbStore</cstring>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QDoubleSpinBox" name="sbStore">
                   <property name="minimumSize">
                    <size>
                     <width>157</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="prefix">
                    <string>C$ </string>
                   </property>
                   <property name="maximum">
                    <double>999999999.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Papeleria</string>
                   </property>
                   <property name="buddy">
                    <cstring>sbPaperWork</cstring>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QDoubleSpinBox" name="sbPaperWork">
                   <property name="minimumSize">
                    <size>
                     <width>157</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="suffix">
                    <string> %</string>
                   </property>
                   <property name="maximum">
                    <double>100.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Transporte</string>
                   </property>
                   <property name="buddy">
                    <cstring>sbTransportation</cstring>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QDoubleSpinBox" name="sbTransportation">
                   <property name="minimumSize">
                    <size>
                     <width>157</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="suffix">
                    <string> %</string>
                   </property>
                   <property name="maximum">
                    <double>100.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <widget class="QCheckBox" name="ckISO">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Aplicar ISO</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="ckTaxes">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Exonerar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Totales de declaración</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinAndMaxSize</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Peso</string>
               </property>
               <property name="buddy">
                <cstring>sbWeight</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Flete </string>
               </property>
               <property name="buddy">
                <cstring>sbFreight</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Seguro</string>
               </property>
               <property name="buddy">
                <cstring>sbInsurance</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Otros Gastos</string>
               </property>
               <property name="buddy">
                <cstring>sbOther</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="sbWeight">
               <property name="minimumSize">
                <size>
                 <width>157</width>
                 <height>0</height>
                </size>
               </property>
               <property name="suffix">
                <string> Kg</string>
               </property>
               <property name="maximum">
                <double>999999999.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="sbFreight">
               <property name="minimumSize">
                <size>
                 <width>157</width>
                 <height>0</height>
                </size>
               </property>
               <property name="prefix">
                <string>US$ </string>
               </property>
               <property name="maximum">
                <double>999999999.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="sbInsurance">
               <property name="minimumSize">
                <size>
                 <width>157</width>
                 <height>0</height>
                </size>
               </property>
               <property name="prefix">
                <string>US$ </string>
               </property>
               <property name="maximum">
                <double>999999999.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="sbOther">
               <property name="minimumSize">
                <size>
                 <width>157</width>
                 <height>0</height>
                </size>
               </property>
               <property name="prefix">
                <string>US$ </string>
               </property>
               <property name="maximum">
                <double>999999999.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="OrderedEditTable" name="tabledetails">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabnavigation">
       <attribute name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/res/table.png</normaloff>:/icons/res/table.png</iconset>
       </attribute>
       <attribute name="title">
        <string notr="true"/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tablenavigation">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;Buscar</string>
            </property>
            <property name="buddy">
             <cstring>txtSearch</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtSearch"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="xdockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Detalles</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTabWidget" name="tabTotalsAccounts">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabaccounts">
        <attribute name="title">
         <string>Movimientos Contables</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QTableView" name="tableaccounts">
           <property name="contextMenuPolicy">
            <enum>Qt::ActionsContextMenu</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabtotals">
        <attribute name="title">
         <string>Totales</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QTableView" name="tabletotals">
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OrderedEditTable</class>
   <extends>QTableView</extends>
   <header>utility.widgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtPolicy</tabstop>
  <tabstop>cbProvider</tabstop>
  <tabstop>dtPicker</tabstop>
  <tabstop>txtSource</tabstop>
  <tabstop>cbWarehouse</tabstop>
  <tabstop>sbAgency</tabstop>
  <tabstop>sbStore</tabstop>
  <tabstop>sbPaperWork</tabstop>
  <tabstop>sbTransportation</tabstop>
  <tabstop>ckISO</tabstop>
  <tabstop>ckTaxes</tabstop>
  <tabstop>sbWeight</tabstop>
  <tabstop>sbFreight</tabstop>
  <tabstop>sbInsurance</tabstop>
  <tabstop>sbOther</tabstop>
  <tabstop>tabledetails</tabstop>
  <tabstop>tabletotals</tabstop>
  <tabstop>tableaccounts</tabstop>
  <tabstop>txtProvider</tabstop>
  <tabstop>txtWarehouse</tabstop>
  <tabstop>txtExchangeRate</tabstop>
  <tabstop>tablenavigation</tabstop>
  <tabstop>txtSearch</tabstop>
  <tabstop>tabTotalsAccounts</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
